backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/PatientManagementTest.java

```diff
@@ -391,64 +391,6 @@ void queryingDeletedPatients_admin_ok() {
         List.of());
   }
 
-  @Test
-  void queryingPatientTestResults_standardUser_ok() throws Exception {
```

Comment:
This is mostly a note for me, but since runtime schema wiring proved difficult to test in isolation, I think this pair of tests was the one thing verifying that entry_only users can't request just any field of the `patient` type.

I can replace these tests with ones targeting a separate restricted access field on a patient query in a future PR.